src/kayenta/edit/nameAndDescription.tsx

```diff
@@ -60,7 +60,7 @@ function mapStateToProps(state: ICanaryState) {
   }
 }
 
-function mapDispatchToProps(dispatch: (action: Action & any) => void) {
+function mapDispatchToProps(dispatch: (action: Action & any) => void): INameAndDescriptionDispatchProps {
```

Comment:
I had originally added some stuff to this file, but later took it out. In the process I discovered these functions weren't actually type checking themselves